modules/lib/src/main/resources/assets/js/app/wizard/page/LiveFormPanel.ts

```diff
@@ -658,8 +581,15 @@ export class LiveFormPanel
     }
 
     setModel(liveEditModel: LiveEditModel) {
+
+        if (this.liveEditModel) {
+            this.liveEditModel?.getSiteModel()?.unApplicationAdded(this.reloadNeededHandler);
+            this.liveEditModel?.getSiteModel()?.unApplicationRemoved(this.reloadNeededHandler);
+        }
```

Comment:
You either use `if (this.liveEditModel)` or `this.liveEditModel?`. Using both doesn't make sense.